### ISSUE TITLE
Listed Docker Compose Down commands don't work for uninstalling volumes?

### ISSUE THREAD

**Issue by Kuiriel** [azerothcore/azerothcore-wotlk#14434]
### Current Behaviour

TL;DR: Docker Compose Down commands appear to be unknown. In the end I just used the Docker Desktop environment where I could click on the trash can symbol for containers, images and volumes to delete it. Maybe the acore-docker guide needs to be updated with a different command?

===

Sorry if I've done something stupid and am wasting your time. I followed the ChromieCraft Docker Desktop install guide, which was sourced from https://www.azerothcore.org/acore-docker/

docker compose ps and other instructions for installing and running docker desktop azeroth core functioned without issue.

I wanted to install modules but the folder names don't match module installation instructions for docker desktop, and the bash dashboard instructions don't appear to indicate how to install modules if not using cmake which is in all the module instructions, so I thought I would remove everything and start over with the classic install guide instead.

I got stuck on the removing everything bit, and feel proper daft.

For some reason I seem to be unable to remove the containers to start fresh. There are listed commands for uninstalling and removing the volumes which windows responds with "unknown command"
```
docker compose down –rmi all -v –remove-orphans
docker compose down –remove-orphans
```

results in either

unknown command "–remove-orphans" for "docker compose down"

or 

unknown command "–rmi" for "docker compose down"

### Expected Blizzlike Behaviour

Issue is not blizzard-like behavior related.

### Steps to reproduce the problem

Install via acore-docker guide.
Attempt to remove via docker compose down –rmi all -v –remove-orphans
See unknown command "–remove-orphans" for "docker compose down"

### AC rev. hash/commit

Shouldn't be needed for this?

### Operating system

Windows 11 x64

**Comment by Kitzunu** [comment]
Make an issue here https://github.com/azerothcore/acore-docker

**Comment by Kuiriel** [comment]
Thank you for the guidance. Sorry for posting in the wrong place and expending your time. Thank you for your work on this beautiful project.

**Comment by Kitzunu** [comment]
That's fine, we can't know everything at first :D
